lib/train/extras/os_common.rb

```diff
@@ -124,6 +129,7 @@ def detect_family_type # rubocop:disable Metrics/CyclomaticComplexity, Metrics/P
       # unix based systems combine the above
       return true if pf == 'unix' and detect_darwin
       return true if pf == 'unix' and detect_esx
+      # This is assuming that pf is set to unix, this should be if pf == 'linux'
```

Comment:
we should remove that command, since from trains perspective linux is a subset of a unix-like operating system.